integrationtests/mayavi/test_contour.py

```diff
@@ -177,6 +177,68 @@ def do(self):
 
         # If we have come this far, we are golden!
 
+    def do_profile(self):
```

Comment:
The intent is to add a framework for memory checks by reusing integration suite. So in that context, it does not check the specifics.